packages/kbn-es/src/install/archive.js

```diff
@@ -107,4 +107,13 @@ async function configureKeystore(installPath, password, log = defaultLog, bundle
     cwd: installPath,
     env,
   });
+
+  await execa(
```

Comment:
__note:__ just for local tests, we won't need to do that once @kbn/es accepts keystore arguments